spring-scaffold-cli/src/main/java/br/com/netodevel/command/template/TemplateHandler.java

```diff
@@ -79,7 +76,8 @@ private ExitStatus generateJmsAwsSQS() {
 
```

Comment:
Use a logger instead of `System.out.println` to print error messages, and consider logging the exception instead of just its message.